string name_a
string alias_a
int64 id
#Nombre del move_group al que pertenece
string name_mg
int64 DOF_mg
pr2_moveit_ltl/limit[] limits
